tests/console/docker.pm

```diff
@@ -108,6 +108,15 @@ sub run {
     $output = script_output('docker container run tw:saved curl -I google.de');
     die("network is not working inside of the container tw:saved") unless ($output =~ m{Location: http://www\.google\.de/});
 
+    # docker run supports --init parameter
+    record_info 'run --init', 'Test if docker run supports --init parameter';
+    assert_script_run 'docker run --rm --init opensuse/tumbleweed ps --no-headers -xo "pid args" | grep "1 /dev/init"';
+
+    # docker has all necessary man pages
+    record_info 'man pages', 'Check if docker has all necessary man pages';
+    assert_script_run 'man -P cat docker build | grep "docker-build - Build an image from a Dockerfile"';
+    assert_script_run 'man -P cat docker config | grep "docker-config - Manage Docker configs"';
```

Comment:
This is not enough to check that the manual is shown properly.
See screenshot http://imagebin.nue.suse.com/2427

Reply:
Can you please describe what is in that screenshot? I'm visually impaired.

Reply:
I was showing the different output of the command man docker config on Leap 42.3, Leap 15 and Tumbleweed.
But I got confused and assert for the title seems good enough. You can ignore my previous comment.